docs/guides/kubernetes/deploy-kubernetes-cluster-using-kubeadm/index.md

```diff
@@ -68,11 +68,11 @@ While kubeadm automates several cluster-provisioning tasks, there are other even
 
 ### Kubernetes Cluster Architecture
 
-A Kubernetes cluster consists of a primary node and secondary nodes. The primary node hosts the *control plane*, which is the combination of all the components that provide it the ability to maintain the desired cluster state. This cluster state is defined by manifest files and the <abbr title="kubectl is a command line tool used to interact with the Kubernetes cluster.">kubectl</abbr> tool. While the control plane components can be run on any cluster node, it is a best practice to isolate the control plane on its own node and to run any application containers on a separate secondary node. A cluster can have a single secondary node or up to 5000. Each secondary node must be able to maintain running containers in a Pod and be able to communicate with the primary node's control plane.
+A Kubernetes cluster consists of a control plane and worker nodes. The control plane hosts the *control plane*, which is the combination of all the components that provide it the ability to maintain the desired cluster state. This cluster state is defined by manifest files and the <abbr title="kubectl is a command line tool used to interact with the Kubernetes cluster.">kubectl</abbr> tool. While the control plane components can be run on any cluster node, it is a best practice to isolate the control plane on its own node and to run any application containers on a separate worker node. A cluster can have a single worker node or up to 5000. Each worker node must be able to maintain running containers in a Pod and be able to communicate with the control plane.
```

Comment:
"The control plane hosts the *control plane*" should be updated

Reply:
Done!!!